tdrs-backend/tdpservice/stts/migrations/0007_stt_ssp.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-10-28 20:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
run_before = [
        ('stts', '0006_alter_stt_filenames'),
    ]

This will solve the problem

Reply:
run_before worked. The docs say to use run_before when it is impractical to reorder your dependencies. I think it would be impractical for us to rollback our migrations on all the servers they are running on.